xbmc/pvr/timers/PVRTimerInfoTag.h

```diff
@@ -120,13 +122,46 @@ namespace PVR
 
     bool IsRecording(void) const { return m_state == PVR_TIMER_STATE_RECORDING; }
 
+    /*!
+      * @brief Checks whether is timer has a timer type. Can be false if
+      *        no PVR client supporting timers is enabled.
```

Comment:
any reason for not just using values? shared_ptr seems unnecessary overhead to me here, and should only be used in case you pass things around not knowing what the lifetime is imo.